extensions/amp-analytics/0.1/vendors.js

```diff
@@ -552,6 +554,85 @@ export const ANALYTICS_CONFIG = /** @type {!JSONType} */ ({
     'optout': '_gaUserPrefs.ioo',
   },
 
+  // 'googleanalytics-legacycookie' is an exact copy of 'googleanalytics'
+  // except that it uses a different cookie name for CLIENT_ID
+  // It's introduced as a temporary solution for #5761.
+  'googleanalytics-legacycookie': {
```

Comment:
My understanding is that being able to read this cookie is not temporary.

Lets call this `-v2` or something like that. Legacy as a word tends to be a function over time, and could be confused when all this is legacy in 2 years.